packages/react-native-editor/babel.config.js

```diff
@@ -1,53 +1,50 @@
 /** @format */
 
-module.exports = function( api ) {
-	api.cache( true );
-	return {
-		presets: [ 'module:metro-react-native-babel-preset' ],
-		plugins: [
-			'@babel/plugin-proposal-async-generator-functions',
-			'@babel/plugin-transform-runtime',
-			[
-				'react-native-platform-specific-extensions',
-				{
-					extensions: [ 'css', 'scss', 'sass' ],
-				},
-			],
-		],
-		overrides: [
+module.exports = {
```

Comment:
Did you try to move those overrides to the root Babel config? I guess it's fine to keep it local for start, but it looks like it doesn't use WordPress preset at all. This might be confusing for contributors.

Reply:
Do you mean to merge both configs together?

Reply:
Yes, ideally there is only one but it might be too much work. Let's skip it for now :)